src/client/Client.cpp

```diff
@@ -1,15 +1,15 @@
-#include <stdlib.h>
+#include <cstdlib>
 #include <iostream>
 #include <sstream>
 #include <string>
 #include <vector>
 #include <map>
 #include <iomanip>
-#include <time.h>
-#include <stdio.h>
+#include <ctime>
+#include <cstdio>
 #include <deque>
 #include <fstream>
-#include <dirent.h>
+#include <cdirent>
```

Comment:
I will push fixes, including this one. cstdint is C++11 and we compile using C++98